From 28d32e9cb656d8df391cb36824e2e0afcbe3e375 Mon Sep 17 00:00:00 2001
From: "Gary D. Gregory" <ggregory@apache.org>
Date: Fri, 17 Feb 2012 23:06:26 +0000
Subject: [PATCH] [IO-303] TeeOutputStream fails executing branch.close() when
 main.close() raised an exception. Use a different impl with < LOCs.

git-svn-id: https://svn.apache.org/repos/asf/commons/proper/io/trunk@1245798 13f79535-47bb-0310-9956-ffa450edef68
---
 src/changes/changes.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/changes/changes.xml b/src/changes/changes.xml
index 87f8646a82e..69318ca7175 100644
--- a/src/changes/changes.xml
+++ b/src/changes/changes.xml
@@ -40,7 +40,7 @@ The <action> type attribute can be add,update,fix,remove.
 
   <body>
     <release version="2.2" date="TBA">
-      <action dev="ggregory" type="add" issue="IO-303" due-to="ggregory">
+      <action dev="ggregory" type="add" issue="IO-303" due-to="fabian.barney">
         TeeOutputStream does not call branch.close() when main.close() throws an exception 
       </action>        
       <action dev="ggregory" type="add" issue="IO-301" due-to="kaykay.unique">
